src/main/java/org/mifos/integrationtest/config/AmsConnectorConfig.java

```diff
@@ -0,0 +1,25 @@
+package org.mifos.integrationtest.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.*;
+
+import javax.annotation.PostConstruct;
+
+@Component
+public class AmsConnectorConfig {
+
+    @Value("${mifos-connector.endpoints.account_status}")
+    public String accountStatusEndpoint;
+
+    @Value("${mifos-connector.contactpoint}")
+    public String amsConnectorContactPoint;
+
+    public String accountStatusUrl;
+
+    @PostConstruct
+    private void setup() {
```

Comment:
instead of adding two fields and then a post constuct method to join them, 
you can create a single @Autowired constructor to do all these in one go

Reply:
I've taken example from repo itself in whole repo it is done like this only

Reply:
We should be able to introduce better practices when they are not available in the code. We don't need to follow bad practices just because they are present in the code base